metaflow/plugins/airflow/airflow.py

```diff
@@ -470,7 +472,22 @@ def _to_job(self, node):
         )
         if AIRFLOW_KUBERNETES_CONN_ID is not None:
             k8s_operator_args["kubernetes_conn_id"] = AIRFLOW_KUBERNETES_CONN_ID
-        else:
+            k8s_operator_args["in_cluster"] = False
+        if AIRFLOW_KUBERNETES_KUBECONFIG_CONTEXT is not None:
+            k8s_operator_args["cluster_context"] = AIRFLOW_KUBERNETES_KUBECONFIG_CONTEXT
+            k8s_operator_args["in_cluster"] = False
+        if AIRFLOW_KUBERNETES_KUBECONFIG_FILE is not None:
+            k8s_operator_args["config_file"] = AIRFLOW_KUBERNETES_KUBECONFIG_FILE
+            k8s_operator_args["in_cluster"] = False
+
+        if all(
+            x is None
+            for x in [
+                AIRFLOW_KUBERNETES_KUBECONFIG_FILE,
+                AIRFLOW_KUBERNETES_KUBECONFIG_CONTEXT,
+                AIRFLOW_KUBERNETES_CONN_ID,
+            ]
+        ):
```

Comment:
won't this be simplified if we set `k8s_operator_args["in_cluster"] = True` above L:473?